app/representers/api/podcast_import_representer.rb

```diff
@@ -8,9 +8,16 @@ class Api::PodcastImportRepresenter < Api::BaseRepresenter
   property :status, writeable: false
   property :created_at, writeable: false
   property :updated_at, writeable: false
+  property :updated_at, writeable: false
+  property :feed, writeable: false
```

Comment:
Is this a string?  (Guess i was just expecting something more like `feed_url`)

Reply:
This is the output of Feedjira: a nested JSON object containing the parsed structure of the podcast rss.  There was some oscillation in design between how to validate the rss (client versus server). So in one timeline, this was used on the client side to display things like images.

Now it acts more as a sentinel and sort of maps to the semantics of the `verify-rss` endpoint: "hay validate this podcast rss and give me the validated data." Could be useful?

Reply:
Gotcha.